docs/source/examples/Natural and artificial perturbations.mystnb

```diff
@@ -44,7 +44,7 @@ pio.renderers.default = "notebook_connected"
 The poliastro package now has several commonly used natural perturbations. One of them is atmospheric drag! See how one can monitor decay of the near-Earth orbit over time using our new module `poliastro.twobody.perturbations`!
 
 ```{code-cell} ipython3
-R = Earth.R.to(u.km).value
+R = Earth.R.to(u.km)
```

Comment:
Were the changes in this notebook intended? They do not introduced any kind of feature nor increase the performance.

Reply:
The doctest builds in this PR yielded an error since I had changed the signature of `LithobrakeEvent` by allowing an astropy `Quantity` object instead of a float. So a corresponding change needed to be made in the doc examples.

Reply:
Alright, so let us make the `Event` class and the members which inherit from it to work without units. This was the original behavior, so let us keep it this way.

Reply:
Only files modified in this PR should be the `events.py` and associated tests.